src/data/github/mod.rs

```diff
@@ -74,3 +76,19 @@ impl GithubRelease {
         None
     }
 }
+
+impl ReleaseArtifacts {
```

Comment:
it is somewhat surprising that this impl is here- i also get why it's here but i wonder if there's not a better way to structure things to make it easier to find